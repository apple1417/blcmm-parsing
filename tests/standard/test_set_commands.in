<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment>set </comment>
			<comment>set a </comment>
			<code profiles="default">set a b </code>
			<code profiles="default">set a b c</code>
			<comment>set <body></comment>
			<code profiles="default">set a b <body></code>
			<code profiles="default">set "/\"' &apos; ><</code>
			<code profiles="default">set <BLCMM v="1"> #<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!> <head> <type offline="false" name="BL2"/> <profiles> <profile current="true" name="default"/> </profiles> </head> <body> <category name="root"> </category> </body> </BLCMM> #Commands:</code>
		</category>
	</body>
</BLCMM>

#Commands:
set 
set a 
set a b 
set a b c
set <body>
set a b <body>
set "/\"' &apos; ><
set <BLCMM v="1"> #<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!> <head> <type offline="false" name="BL2"/> <profiles> <profile current="true" name="default"/> </profiles> </head> <body> <category name="root"> </category> </body> </BLCMM> #Commands:

