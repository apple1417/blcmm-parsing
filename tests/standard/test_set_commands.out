<BLCMM v="1">
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="default" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="root">
			<comment>set </comment>
			<comment>set a </comment>
			<code profiles="default">set a b </code>
			<code profiles="default">set a b c</code>
			<comment>set &lt;body&gt;</comment>
			<code profiles="default">set a b &lt;body&gt;</code>
			<code profiles="default">set &quot;/\&quot;&apos; &amp;apos; &gt;&lt;</code>
			<code profiles="default">set &lt;BLCMM v=&quot;1&quot;&gt; #&lt;!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!&gt; &lt;head&gt; &lt;type offline=&quot;false&quot; name=&quot;BL2&quot;/&gt; &lt;profiles&gt; &lt;profile current=&quot;true&quot; name=&quot;default&quot;/&gt; &lt;/profiles&gt; &lt;/head&gt; &lt;body&gt; &lt;category name=&quot;root&quot;&gt; &lt;/category&gt; &lt;/body&gt; &lt;/BLCMM&gt; #Commands:</code>
		</category>
	</body>
</BLCMM>
